workflow-job

```diff
@@ -597,6 +617,9 @@ flows["${sha1}"] = {
             def tagName = tag();
             sh "docker login -u ${dUser} -p ${dPassword}"
 
+            sh "docker tag ${images.jamesCassandraRabbitMQLdap} linagora/james-rabbitmq-ldap-project:${tagName}"
```

Comment:
by existing convention, I prefer to have a `linagora/james-cassandra-rabbitmq-ldap-project` image